src/Microsoft.IdentityModel.Tokens/Encryption/AuthenticatedEncryptionProvider.cs

```diff
@@ -239,29 +239,30 @@ internal SymmetricSignatureProvider CreateSymmetricSignatureProvider()
 
         /// <summary>
-        /// Encrypts the 'plaintext'
+        /// Encrypts the specified plaintext.
         /// </summary>
-        /// <param name="plaintext">the data to be encrypted.</param>
-        /// <param name="authenticatedData">will be combined with iv and ciphertext to create an authenticationtag.</param>
-        /// <returns><see cref="AuthenticatedEncryptionResult"/>containing ciphertext, iv, authenticationtag.</returns>
-        /// <exception cref="ArgumentNullException">plaintext is null or empty.</exception>
-        /// <exception cref="ArgumentNullException">authenticationData is null or empty.</exception>
-        /// <exception cref="SecurityTokenEncryptionFailedException">AES crypto operation threw. See inner exception for details.</exception>
+        /// <param name="plaintext">The data to be encrypted.</param>
+        /// <param name="authenticatedData">Will be combined with the initialization vector and ciphertext to create an authentication tag.</param>
+        /// <returns>A <see cref="AuthenticatedEncryptionResult"/> the containing ciphertext, initialization vector, the and authentication tag.</returns>
+        /// <exception cref="ArgumentNullException">Thrown if <paramref name="plaintext"/> is null or empty.</exception>
+        /// <exception cref="ArgumentNullException">Thrown if <paramref name="authenticationData"/> is null or empty.</exception>
+        /// <exception cref="SecurityTokenEncryptionFailedException">Thrown if AES crypto operation threw. See inner exception for details.</exception>
+        /// <exception cref="ObjectDisposedException">Thrown if the internal <see cref="SignatureProvider"/> is disposed.</exception>
         public virtual AuthenticatedEncryptionResult Encrypt(byte[] plaintext, byte[] authenticatedData)
         {
             return Encrypt(plaintext, authenticatedData, null);
         }
 
         /// <summary>
-        /// Encrypts the 'plaintext'
+        /// Encrypts the specified plaintext.
         /// </summary>
-        /// <param name="plaintext">the data to be encrypted.</param>
-        /// <param name="authenticatedData">will be combined with iv and ciphertext to create an authenticationtag.</param>
-        /// <param name="iv">initialization vector for encryption.</param>
-        /// <returns><see cref="AuthenticatedEncryptionResult"/>containing ciphertext, iv, authenticationtag.</returns>
-        /// <exception cref="ArgumentNullException"><paramref name="plaintext"/> is null or empty.</exception>
-        /// <exception cref="ArgumentNullException"><paramref name="authenticatedData"/> is null or empty.</exception>
-        /// <exception cref="SecurityTokenEncryptionFailedException">Thrown if the AES crypto operation threw. See inner exception for details.</exception>
+        /// <param name="plaintext">The data to be encrypted.</param>
+        /// <param name="iv">The initialization vector used for the encryption algorithm.</param>
```

Comment:
perhaps: 'vector used with the encryption'

Reply:
Done